build/broccoli/build-tests.js

```diff
@@ -24,14 +24,43 @@ module.exports = function(packages) {
     includeGlimmerAMD(amdTree),
     includeTestFiles(amdTree),
     includeVendorDependencies(),
-    includeTestHarness()
+    includeTestHarness(),
+    includeNodeTests()
   ]);
 
   return funnel(testTree, {
     destDir: 'tests'
   });
 }
 
+function includeNodeTests() {
+  let tsTree = funnel('packages', {
+    include: ['@glimmer/**/*.ts'],
```

Comment:
@stefanpenner Not sure if I understand your question exactly. This includes all `.ts` files but not `.d.ts` files (which match `.ts`), so any non-TS files aren't included. What do you mean by "non-test files"?